src/OnboardingSPA/components/SocialMediaForm/index.js

```diff
@@ -4,8 +4,7 @@ import { useState, useEffect } from '@wordpress/element';
 
```

Comment:
Not sure if you guys were aware of this @officiallygod and @Yashita101 before approving but when I empty any input field I immediately get an error on my console 
[screen-capture.webm](https://user-images.githubusercontent.com/38878906/210197071-0f92ee80-698c-484f-9d01-fc917b223866.webm)

Reply:
Yeah so that error is coming from file SocialMediaForm setDataAndActiveErrorState function looks like the existing logic there had that bug let me fix that too here.

Reply:
Fixed.